recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set name = "whatlies" %}
+{% set version = "0.6.5" %}
+
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/whatlies-{{ version }}.tar.gz
+  sha256: 0b97c2b5b0bbc5c4044278ea8b2d3222cbb125e1fbf7bd854bb7f09b557e13d7
+
+build:
+  number: 0
+  noarch: python
+requirements:
+# ref: https://gitter.im/conda-forge/conda-forge.github.io?at=6195afbdb5ba9e5a11c02791
+  host:
+    - python >=3.6,<3.8
+  run:
+    - python >=3.6,<3.8
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv
+      run_exports:
+        - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
+    requirements:
+      host:
+        - pip
+        - python >=3.6,<3.8
+      run:
+        - python >=3.6,<3.8
+        # base_packages
+        - altair >=4.0.1
+        - bpemb >=0.3.0
+        - gensim ~=3.8.3
+        - matplotlib-base >=3.2.0
+        - scikit-learn >=0.24.0
+      run_constrained:
+        # umap_packages
+        - umap-learn >=0.4.0
+        # rasa_packages
+        - rasa >=2.7.0
+        # fasttext_packages
+        - fasttext ~=0.9.1
+        # spacy_packages
+        - spacy >=3.0.1
+        - spacy-lookups-data >=0.3.2
+        # s2v_packages (+= spacy_packages)
+        - sense2vec >=1.0.2
+        # tf_packages
+        - tensorflow >=2.3.0,<2.5
+        - tensorflow-text >=2.3.0
+        - tensorflow-hub >=0.8.0
+        # transformers_dep
+        - transformers >=4.3.0
+        # sentence_tfm_dep
+        - sentence-transformers >=0.3.8
+    test:
+      imports:
+        - whatlies
+      commands:
+        - pip check
+      requires:
+        - pip
+        # test_packages
+        #- torch >=1.6.0
+        #- torch
+        # - flake8 >=3.6.0
+        # - pytest >=4.0.2
+        # - black >=19.3b0
+        # - pytest-cov >=2.6.1
+        # - nbval >=0.9.5
+        # - pre-commit >=2.2.0
```

Comment:
Normally, most test packages that upstream lists aren't required because you don't need coverage etc. I'd start with pytest and maybe torch and then see if I get a _missing dependency_ error.

Reply:
Thank you. I was reading the conda-forge docs and realized I did not need those test-packages. So, removed them.